apps/blockchain/config/dev.secret.exs.example

```diff
@@ -1,2 +1,6 @@
 use Mix.Config
-config :ethereumex, url: "https://mainnet.infura.io/<your api key here>"
+
+config :ethereumex,
+  url: "https://mainnet.infura.io/<your api key here>",
+  ipc_path: "/.local/share/io.parity.ethereum/jsonrpc.ipc",
```

Comment:
Will this path work on OSX? Assuming parity is installed via home-brew?

Reply:
yep, it should work. parity's file should be locate in "~/.local/share/io.parity.ethereum/jsonrpc.ipc"
ipc server is started by default